src/components/player-area/player-area-provider.js

```diff
@@ -82,7 +82,7 @@ class PlayerAreaProvider extends Component {
   _validateComponentData = componentData => {
     // we keep option `container` for backward compatibility. documentation are showing `area` property
     const hasAreaProperty = componentData.container || componentData.area;
-    if (!componentData.get || !componentData.presets || !hasAreaProperty) {
+    if ((!componentData.get && !componentData.removeComponent) || !componentData.presets || !hasAreaProperty) {
```

Comment:
change the log below accordingly

Reply:
Current message also describes our case. If we need to make it more specific then we need to separate the if statement